server/index.js

```diff
@@ -38,13 +38,15 @@ const createNotificationJobs = require('./services/jobs/notificationJobs')
 const createDeliusClient = require('./data/deliusClient')
 const createProbationTeamsClient = require('./data/probationTeamsClient')
 const createRoService = require('./services/roService')
+const createCaService = require('./services/caService')
 
 const signInService = createSignInService()
 const licenceService = createLicenceService(licenceClient)
 const conditionsService = createConditionsService(config)
 const deliusClient = createDeliusClient(signInService)
 const probationTeamsClient = createProbationTeamsClient(signInService)
 const roService = createRoService(deliusClient, nomisClientBuilder)
+const caService = createCaService(deliusClient, nomisClientBuilder)
```

Comment:
so you would pass through `roService` here

Reply:
ok

Reply:
ok